groups_admin.rb

```diff
@@ -68,22 +72,43 @@
   def status_data
     {:current_time => Time.now.iso8601}
   end
-end
 
-def update_accept_header(extension,mime_type)
-  if request.url.match(/.#{extension}$/)
-    request.accept.unshift(mime_type)
-    request.path_info = request.path_info.gsub(/.#{extension}$/, '')
+  def update_accept_header(extension, mime_type)
+    if request.url.match(/.#{extension}$/)
```

Comment:
Rather than regular expressions, since they can be tricky and generally use more resources, why not just look for the extension at the end of the string?